themes/default/content/blog/cloud-engineering-fuels-startup-innovation/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Cloud engineering fuels the next chapter of startup innovation"
+
```

Comment:
Our Hugo implementation has a title length of 60 characters, and our linter checks for that. A new flag was added that allows a longer title, so you'll need that flag in your metadata (the top part here between the `---`) to make longer titles. More info at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md#make-a-new-post under "Important" if you want it.
```suggestion
allow_long_title: True
```